.gitignore

```diff
@@ -11,3 +11,6 @@ tsconfig.tsbuildinfo
 /wordpress
 packages/php/*/vendor
 tests_env/vendor
+doc/active_ticket.md
+run-tests.sh
+fix_bom.php
```

Comment:
I think for personal workflows and "hack"-files, I would try to configure a local gitignore file, e.g.: https://dev.to/fronkan/a-personal-gitignore-even-for-a-single-repository-4o7h

Reply:
But not a blocker to me atm